corehq/apps/cloudcare/templates/form_entry/templates.html

```diff
@@ -6,7 +6,6 @@
         class: elementTile,
         css: {
           'gr-no-children': $data.children().length === 0,
-          'gr-has-no-questions': $data.children().every(function(d) { return d.type() !== 'grouped-element-tile-row' }),
```

Comment:
I had to think pretty hard because of the double negatives but it seems like you're right that it's just a subset. I did notice an error with `hasAnyNestedQuestions ` and made the fix [here](https://github.com/dimagi/commcare-hq/pull/34220)

Reply:
I don't fully follow the logic but I agree that it seems unnecessary, and leftover from the past strategy of removing margins to "hide" elements.